docs/src/main/asciidoc/logging.adoc

```diff
@@ -25,7 +25,7 @@ You can use any of the <<logging-apis,following APIs>>:
 [[jboss-logging]]
 == Use JBoss Logging for application logging
 
-No additional dependencies are needed when using the JBoss Logging API; it is automatically provided.
+Using the JBoss Logging API, your application requires no additional dependencies, as Quarkus automatically provides it.
```

Comment:
```suggestion
When using the JBoss Logging API, your application requires no additional dependencies, as Quarkus automatically provides it.
```

Otherwise it doesn't look like a sentence to me?